apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts

```diff
@@ -359,8 +362,19 @@ export class UpsertWorkflowUseCase {
             },
           })
         );
-        let htmlBody = (result.preview as EmailRenderOutput).body ?? '';
-        htmlBody = this.removeBrandingFromHtml(htmlBody);
+        let htmlBody = this.removeBrandingFromHtml((result.preview as EmailRenderOutput).body ?? '');
+        try {
+          htmlBody = await format(htmlBody, {
+            parser: 'html',
+            printWidth: 120,
+            tabWidth: 2,
+            useTabs: false,
+            htmlWhitespaceSensitivity: 'css',
+          });
+        } catch (error) {
+          this.logger.warn({ err: error }, 'Failed to prettify HTML');
+        }
```

Comment:
try to format the code, if it fails, just use the unformatted code, don't fail the upsert